importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -1105,30 +1106,34 @@ static void addFermataToChord(const Notation& notation, ChordRest* cr)
       else
             na->setPlacement(na->propertyDefault(Pid::PLACEMENT).value<Placement>());
       setElementPropertyFlags(na, Pid::PLACEMENT, direction);
-      if (cr->segment() == nullptr && cr->isGrace())
-            cr->el().push_back(na);       // store for later move to segment
+      if (!seg) {
```

Comment:
here I'd prefer
```cpp
      if (!seg == nullptr && cr->isGrace())
            cr->el().push_back(na);       // store for later move to segment
```

Reply:
Won't work though, the entire if/else needs to be skipped?

Reply:
what if seg is invalid but is not grace?  then it still attempts to dereference seg in the next line. But i agree about returning on !seg a little later if you'd like

Reply:
Yes, we need to handle seg here twice

Reply:
And may need to loose the `const`

Reply:
Looks like const was causing build issues, even though at first seemed like a good idea. Rebased.
(P.S. Lose for drop/remove - "double o seven" loose or loosen for untying a knot :)

Reply:
Ah, thanks, I never get my head round that difference ;-)